build-system/server/new-server/transforms/transform.ts

```diff
@@ -19,15 +19,32 @@ import minimist from 'minimist';
 import posthtml from 'posthtml';
 import transformModules from './modules/modules-transform';
 import transformScriptPaths from './scripts/scripts-transform';
-import transformStories from './stories/stories-transform';
 import transformCss from './css/css-transform';
 
 const argv = minimist(process.argv.slice(2));
-const transforms = [transformStories(), transformScriptPaths(), transformCss()];
+const FOR_TESTING = argv._.includes('integration');
+// Use 9876 if running integration tests as this is the KARMA_SERVER_PORT
+const PORT = FOR_TESTING ? 9876 : 8000;
+const ESM = !!argv.esm;
+
+const transforms = [
+  transformScriptPaths({
+    esm: ESM,
+    port: PORT,
+    fortesting: FOR_TESTING,
+  }),
```

Comment:
Instead of repeating the options object, maybe declare just the object as a constant, so you can reuse it inside `transform()`?

Reply:
done.